src/Core/Types/Types/Scalars/Scalars.cs

```diff
@@ -82,6 +84,8 @@ public static class Scalars
                     typeof(NameType), TypeContext.None) },
                 { ScalarNames.PaginationAmount, new ClrTypeReference(
                     typeof(PaginationAmountType), TypeContext.None) },
+                { ScalarNames.ByteArray, new ClrTypeReference(
+                    typeof(ByteArrayType), TypeContext.None) },
            };
 
         private static readonly Dictionary<Type, ValueKind> _scalarKinds =
```

Comment:
@michaelstaib not sure if byte[] needs to be added here.

Reply:
Adding it here will add this as a default resolution meaning that the schema will now resolver byte arrays automatically to the type `ByteArray`. So, this should be fine.